package com.accenture.bootcamp.exceptions.exercise21;

public class Exercise21 {

    // FIX NPE
    // handle array out of bounds exception
    // call a method which has throws checked exception
    // add catch division by zero exception
    public static void main(String[] args) {



    }
}
